### PR TITLE
 feat(controller): movies controller and tests

### PR DESCRIPTION
**Pull request by robintsunny**
what: movie controller and tests
why: determine where to send requests for POST /movies

### CODE REVIEW

File: lib/plugins/features/movies/controller.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Movie = require('../../../models/movie');
+
+exports.create = (payload) => {
+  return new Movie().save(payload)
+  .then((movie) => new Movie({ id: movie.id }).fetch());
```
Comment (lvamaral) [inline lib/plugins/features/movies/controller.js:7]: nit: you should be able to just return `new Movie().save(payload)`

**Review by lvamaral — APPROVED** [review]

**Comment by lvamaral** [comment]
Not sure how you've done it in other PRs but I'd still practice adding a description like 
```
What: ...
Why: ... 
```
to help frame how you think of PRs

File: lib/plugins/features/movies/controller.js (hunk 1 above)
Comment (robintsunny) [reply lib/plugins/features/movies/controller.js:7]: jessica mentioned that Movie().save() returns a promise and that it was just the syntax of bookshelf that did it this way. do you know more about this?

Comment (jesscxu) [reply lib/plugins/features/movies/controller.js:7]: oh wait, i think im wrong... i thought we had to save the movie and that does not return a model so we have to fetch it again. but actually i think you can do you what lucas said

**Review by jesscxu — APPROVED** [review]
LGTM ... commit message should be something more like `feat(movies): add controller`... there are multiple controllers but only one movie plugin so the scope is movies

File: lib/plugins/features/movies/controller.js (hunk 1 above)
Comment (lvamaral) [reply lib/plugins/features/movies/controller.js:7]: `.save()` returns the Bookshelf object once the promise completes, so you wouldn't even need to refetch the movie if you wanted to do anything with it. If you return it in this response the promise completes; so if you make that change and run the tests again they should still pass

**Pull request merged**